src/components/AddItem.js

```diff
@@ -24,8 +26,24 @@ const AddItem = ({ token }) => {
         });
   };
 
-  const handleSubmit = (event) => {
+  const handleSubmit = async (event) => {
     event.preventDefault();
+    const itemNameInput = formData.itemName;
+
+    // check to see if the itemName already exists in the user's list - convert input to lowercase and remove any punctuation & white space
```

Comment:
I appreciate the comments explaining what is going on here

Reply:
Yes... 💯 %